src/display.js

```diff
@@ -0,0 +1,75 @@
+const displayCommentinfo = (movie, receiveComment, counter) => {
+  const displayComments = (el) => {
+    const ul = document.querySelector('.user-comment');
+    el.forEach((item) => {
+      const li = document.createElement('li');
+      const date = `${item.creation_date}`;
+      li.textContent = `${date}  ${item.username} : ${item.comment}`;
+      ul.appendChild(li);
+    });
+  };
+
+  const commentsdis = (el) => {
+    const ul = document.querySelector('.user-comment');
+    const li = document.createElement('li');
+    const date = `${el.creation_date}`;
+    li.textContent = `${date}  ${el.username} : ${el.comment}`;
+    ul.appendChild(li);
+  };
+  const comments = document.querySelector('.popu');
+  comments.addEventListener('click', (event) => {
+    if (event.target.matches('button')) {
+      event.preventDefault();
+      const username = document.querySelector('.input').value;
+      const comment = document.querySelector('.texarea').value;
+      const itemId = movie.id;
+      const d = new Date();
+      // eslint-disable-next-line camelcase
```

Comment:
- [ ] Kindly remove all linter disables. Linters enforces code quality and therefore should not be disabled. Instead, you should correct the issues raised.

Reply:
Addressed 👍🏼